Directory.Packages.props

```diff
@@ -1,35 +1,35 @@
 <Project>
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-    <RuntimeVersion9>9.0.0</RuntimeVersion9>
```

Comment:
That is already done in another PR 🙃

Please, create a separate PR when you want to update the lib version: that is not required for Button.Label